ignite/README.md

```diff
@@ -0,0 +1,346 @@
+<!--
+Copyright (c) 2015 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+# Apache Ignite
```

Comment:
this looks like the README from the ignite project, which I think is too broad for use here.

You should give links or basic steps for setting up ignite, prepping to run YCSB, and any caveats about this specific binding. For example, the fact that folks need to have their cache named 'usertable' should be noted.

Check out the [Accumulo 1.8 README](https://github.com/brianfrankcooper/YCSB/blob/0.14.0/accumulo1.8/README.md) as an example.

Reply:
Updated